hack/make-rules/test-integration.sh

```diff
@@ -68,10 +68,12 @@ runTests() {
   kube::log::status "Running integration test cases"
 
   KUBE_RACE="-race"
+  # KUBE_TEST_ARGS may contain '$' character which has special meaning in make,
+  # pass it through process environment instead.
+  KUBE_TEST_ARGS="${KUBE_TEST_ARGS:-} ${SHORT:--short=true} --vmodule=${KUBE_TEST_VMODULE} --alsologtostderr=true" \
```

Comment:
Doing it this way I think will pose a problem if `runTests()` runs more than once (such as the loop at L107). `KUBE_TEST_ARGS` will append these each time, because the variable is not local to the function.

Reply:
`KUBE_TEST_ARGS=...` precedes `make ... by using `\`, it only sets environment for `make` command and will not pollute global `KUBE_TEST_ARGS` variable.

Reply:
Ahh okay, I see that. Can you please modify the indentation of the `make` command itself to indicate that more clearly?

Reply:
@cblecker Sure, done and squashed.